GitHubLabels.ps1

```diff
@@ -565,6 +628,121 @@ function Set-GitHubLabel
     }
 }
 
+function Add-GitHubLabel
```

Comment:
What's the common scenario?  Let's say I want to add a label to something.  Does that require a call to New-GithubLabel and Add-GithubLabel?

Reply:
Adding new labels is uncommon.  Adding labels to issues is common and frequent.  I think that separating this functionality out between two separate methods makes sense.  It does raise the question on why we'd then have a merged `Remove-GitHubLabel`.  From a naming perspective, it makes sense in both scenarios, but it is I guess odd that we'd have a converged 'Remove' but a separated 'Add'/'New'

Reply:
@HowardWolosky I agree, would it make more sense to just have two separate functions called Add-GitHubIssueLabel and Remove-GitHubIssueLabel? Does it also make more sense to move these in with the rest of the Issues code?

Reply:
I _think_ we should stick with `Add-GitHubLabel` and `Remove-GitHubLabel`.  Otherwise, if we followed this newer suggestion, we'd _also_ need `Remove-GitHubMilestonLabel` and that just "feels" weird.